### DISCUSSION TITLE
Arquetype Capacitor integración with Angular

### DISCUSSION THREAD

**Discussion by achacofe** [ionic-team/capacitor#5479] [Ideas]
Good morning, I am dedicated to the development of hybrid applications, I previously worked with Ionic.
I am currently in a Franework Angular team at Mercadona, in which they have created an archetype and monorepo to offer developers the structure, components and others so that they can start working on a solid base.
I am conducting research on capacitors to see if it is necessary to create a capacitor archetype, since Mercadona has made the decision to apply the hybrid mobility layer and we have decided to use Capacitor.

I would like to have a conversation to resolve certain doubts about it.
Would it be possible?

Thank you very much in advance,
Greetings!

**Comment by jcesarmobile** [comment]
Capacitor can be used with any framework, Angular included.

We have this short introduction about it
https://capacitorjs.com/solution/angular

Feel free to ask about any questions you have.
